tests/ttiex355p.py

```diff
@@ -10,8 +10,14 @@
 DEVICE_PREFIX = "TTIEX355P_01"
 DEVICE_NAME = "tti355"  # similar enough device that we can use the same emulator
 
-AMPSTOGAUSS = 5
-MAX_FIELD = 10
+AMPSTOGAUSS = 7.278
+MAX_FIELD = 25
+
+# Taken from manual Specification of OUTPUT page 3^M
+MIN_VOLTAGE = 0.0
+MIN_CURRENT = 0.01
+MAX_VOLTAGE = 35.0
+MAX_CURRENT = 5.0
```

Comment:
If these are inherent to the device then I think they should just be set in the emulator as default rather than via the backdoor.

Reply:
They are inherent to the ttiex355p but not the tti355, as such I have moved them into a separate emulator that mostly sub-classes classes of the tti355 with the addition of min voltage and current checks

Reply:
Have kept the constants here for use in the tests elsewhere